src/Core/Localization/CLDR/LocaleData.php

```diff
@@ -368,4 +383,24 @@ public function setCurrencies($currencies)
 
         return $this;
     }
+
+    /**
+     * @return string[]
```

Comment:
Maybe add a comment to define the format, from the Reader code it seems to be an array indexed by `type` ? What does the type attribute refer to?